CHANGES.rst

```diff
@@ -1,6 +1,9 @@
 0.4.3 (unreleased)
 ------------------
- - none yet
+ - Refactor spectral smoothing tools to allow parallelized application *and*
+   memory mapped output (to avoid loading cube into memory).  Created
+   ``apply_function_parallel_spectral`` to make this general.
+   (https://github.com/radio-astro-tools/spectral-cube/pull/474)
```

Comment:
Note here if we're introducing joblib as a dependency.

Reply:
I'd prefer to add it to the REQUIREMENTS so that it gets pip installed w/spectral-cube

Reply:
Agreed. I thought it would be good to note where it was introduced here.